### PR TITLE
[API] `getNodeConfig` API returns a series of initial node configuration

### PR DESCRIPTION
**Pull request by hyunsooda**
configuration

## Proposed changes

`getNodeConfig` API is added in `klay` namespace. It returns a series of node configurations except for some fields that have directory pathsinformation.

```shell
> klay.getNodeConfig
{
  AnchoringPeriod: 1,
  AutoRestartFlag: true,
  DBType: "LevelDB",
  DaemonPathFlag: "",
  DisableUnsafeDebug: false,
  DocRoot: "",
  DownloaderDisable: false,
  ...
  LevelDBBufferPool: true, 
  LevelDBCacheSize: 768,
  LevelDBCompression: 0, 
  LivePruning: false,
  LivePruningRetention: 172800,
  NetworkId: 5000,
  ...
  TrieNodeCacheConfig: {
   CacheType: "LocalCache",
   FastCacheFileDir: "",
   FastCacheSavePeriod: 0,
   LocalCacheSizeMiB: 1234,
   NumFetcherPrefetchWorker: 32,
   RedisClusterEnable: false,
   RedisEndpoints: null,
   RedisPublishBlockEnable: false,
   RedisSubscribeBlockEnable: false,
   UseSnapshotForPrefetch: false
   }, 
   ...
}
```

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Bugfix
- [ ] New feature or enhancement
- [ ] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [ ] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn/blob/master/CONTRIBUTING.md) doc
- [ ] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn)
- [ ] Lint and unit tests pass locally with my changes (`$ make test`)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Related issues

- Please leave the issue numbers or links related to this PR here.

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by blukat29** [comment]
Since the configuration may contain sensitive information, how about moving it under `admin` namespace or making it an IPC-only API?

**Comment by hyunsooda** [comment]
Relocated to `admin` namespace and removed path hiding.

**Review by blukat29 — APPROVED** [review]

File: console/web3ext/web3ext.go [hunk 1]
```diff
@@ -510,6 +510,10 @@ web3._extend({
 			name: 'spamThrottlerConfig',
 			getter: 'admin_spamThrottlerConfig'
 		}),
+		new web3._extend.Property({
```
Comment (JayChoi1736) [inline console/web3ext/web3ext.go:513]: if it is Property, admin_nodeconfig format seems to be more contextual, 
or changes API as web3._extend.Method

Comment (JayChoi1736) [reply console/web3ext/web3ext.go:513]: with this, i think java console is expected to show a function in the form of getGetNodeConfig

Comment (hyunsooda) [reply console/web3ext/web3ext.go:513]: Thank you. Renamed to `nodeConfig`.

**Review by blukat29 — APPROVED** [review]

**Review by yoomee1313 — APPROVED** [review]

**Review by JayChoi1736 — APPROVED** [review]

**Pull request merged**